mcs_benchmark_data/pipelines/commonsense_qa/commonsense_qa_benchmark_extractor.py

```diff
@@ -2,35 +2,23 @@
 
 from mcs_benchmark_data.path import DATA_DIR_PATH
 
+from mcs_benchmark_data.pipelines.commonsense_qa.commonsense_qa_benchmark_file_names import (
+    CommonsenseQaBenchmarkFileNames,
+)
+
 
 class CommonsenseQaBenchmarkExtractor(_Extractor):
     def __init__(
         self,
-        dev_jsonl_file_name: str,
-        test_jsonl_file_name: str,
-        train_jsonl_file_name: str,
-        **kwds
+        file_names: CommonsenseQaBenchmarkFileNames,
```

Comment:
With this change to file_names, you don't really need benchmark-specific extractors any more, since they'll all look the same (take file_names, return it via extract). In effect you've moved the benchmark specialization to the *FileNames class hierarchy. You could call this class something like FileBenchmarkExtractor, or maybe just BenchmarkExtractor, since we don't have any non-file cases that I'm aware of.